test/kind-config.yaml

```diff
@@ -3,3 +3,20 @@ apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
   - role: control-plane
   - role: worker
+kubeadmConfigPatches:
```

Comment:
could we get a comment for these patches please to make it easier for future code readers to quickly know the context?